app/views/spree/shared/_guided_tour.html.erb

```diff
@@ -1,5 +1,5 @@
 <div style="display:none;" class="tour-widget <%= "open" if params[:guided_tour] %>">
-  <% section = $admin ? "backend" : "frontend" %>
+  <% section = @is_admin ? "backend" : "frontend" %>
   <% tour_text = TOUR_TEXT.dig(section, controller_name, action_name) || [] %>
```

Comment:
Slightly OT, but a good alternative would be to use [`controller_path`](https://api.rubyonrails.org/v5.2.1/classes/AbstractController/Base.html#method-c-controller_path) instead and have the tour file keys be like:

```yaml
spree/frontend/home:
  index:
    solidus_guided_tour: #Title of the section of the tour - not displayed to user
      button_action: next #use "next" when there are more than one items on the same page
      title: Solidus Guided Tour #The user-visible tour title
      body_text: |+
        Welcome to the Solidus Guided Tour. We'll take you through some of what we think makes Solidus special.

spree/frontend/taxons:
  show: …
spree/frontend/products:
  show: …
```

Reply:
Ahh, yeah, that works much better! Thanks, I'll pop that in.